lib/missing_step_error.ex

```diff
@@ -0,0 +1,97 @@
+defmodule MissingStepError do
+  @moduledoc """
+  Raises an error, because a feature step is missing its implementation.
+
+  The message of the error will give the user a useful code snippet where
+  variables in feature steps are converted to regex capture groups.
+  """
+
+  defexception [:message]
+
+  @number_regex ~r/(^|\s)\d+(\s|$)/
+  @single_quote_regex ~r/'[^']+'/
+  @double_quote_regex ~r/"[^"]+"/
+
+
+"""
+
+iex(1)> string = "I set my name to \"Dan\" and also \"John\" "
+"I set my name to \"Dan\" and also \"John\" "
+iex(2)> regex = ~r/"[^"]+"/
+~r/"[^"]+"/
+iex(3)> Regex.split(regex, string)
+["I set my name to ", " and also ", " "]
+iex(4)> res = Regex.split(regex, string)
+["I set my name to ", " and also ", " "]
+iex(5)> Enum.reduce(res, "", fn(x, acc) -> acc <> "\"(?<>)\"" <> x end)
+"\"(?<>)\"I set my name to \"(?<>)\" and also \"(?<>)\" "
+iex(6)> Regex.split(~r/3425345435/, string)
+["I set my name to \"Dan\" and also \"John\" "]
+iex(7)> string = "34 hello 45"
+"34 hello 45"
+iex(8)> regex2 = ~r/(^|\s)\d+(\s|$)/
+~r/(^|\s)\d+(\s|$)/
+iex(9)> Regex.split(regex2, string)
+["", "hello", ""]
+iex(10)>
```

Comment:
it was an intended commit, trying out using `Regex.split/2` to add regex capture groups to the string and wanted to carry on with it at home